server/channels/db/migrations/postgres/000142_create_content_flagging_tables.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS ContentFlaggingCommonReviewers (
+	userid VARCHAR(26) PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS ContentFlaggingTeamSettings (
+	teamid VARCHAR(26) PRIMARY KEY,
+	enabled BOOLEAN
+);
+
+CREATE TABLE IF NOT EXISTS ContentFlaggingTeamReviewers (
+	teamid VARCHAR(26),
+	userid VARCHAR(26),
+	PRIMARY KEY (teamid, userid)
+);
+
+CREATE INDEX IF NOT EXISTS idx_contentflaggingteamreviewers_userid ON ContentFlaggingTeamReviewers (userid);
```

Comment:
Can we do this concurrently? It doesn't matter on first creation because it's a new table, but if in the future for some reason we need to re-run the migration and the tables exist but the index does not, it will come in handy.

```suggestion
CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_contentflaggingteamreviewers_userid ON ContentFlaggingTeamReviewers (userid);
```

Reply:
Done. I had to extract the index creation into a separate migration as otherwise morph failed to run the migrations in tests.

Reply:
Huh, why? :thinking: @isacikgoz, any hint?

Reply:
No idea, need to see logs :)